.env.example

```diff
@@ -1,49 +1,49 @@
-MO_PORT=8080
-MO_UPSTREAMCACHEEXPIRATION=10000
-MO_DOWNSTREAMCACHEEXPIRATION=120000
-MO_INITIALMAXDELAY=1700
+#MO_PORT=8080
+#MO_UPSTREAMCACHEEXPIRATION=10000
+#MO_DOWNSTREAMCACHEEXPIRATION=120000
+#MO_INITIALMAXDELAY=1700
 
 # Config
-MO_CONFIG=./config-example.json
+#MO_CONFIG=./config-example.json
```

Comment:
Maybe keep this one un-commented to make it work when init?

Reply:
Or better: remove the comment on init

Reply:
Should be `config.json` according to the init function

Reply:
So: keep it like that in example, but at the time you create the config.json, make sure that this line is updated to:

    MO_CONFIG=./config.json 

in the newly created `.env`

Reply:
Done